src/main/java/org/opentripplanner/routing/edgetype/Timetable.java

```diff
@@ -139,6 +139,8 @@ public TripTimes getNextTrip(State s0, ServiceDay serviceDay, int stopIndex, boo
         /* Search at the state's time, but relative to midnight on the given service day. */
         int time = serviceDay.secondsSinceMidnight(s0.getTimeSeconds());
         // NOTE the time is sometimes negative here. That is fine, we search for the first trip of the day.
+        // Adjust for possible boarding time TODO: This should be included in the trip and based on GTFS
+        time += s0.getOptions().getBoardTime(this.pattern.mode);
```

Comment:
That is true. I'll change that here and also in TransitBoardAlight. I just wanted to check that that you also think these are the most suitable places to add the time. A second option would be to create specific board/alight legs for these by adding a state for that.